zerver/models.py

```diff
@@ -1278,7 +1289,11 @@ class Subscription(ModelReprMixin, models.Model):
     active = models.BooleanField(default=True) # type: bool
     in_home_view = models.NullBooleanField(default=True) # type: Optional[bool]
 
-    DEFAULT_STREAM_COLOR = u"#c2c2c2"
+    PERMISSION_FLAGS = ['can_read', 'can_write', 'can_moderate',
+                        'can_add_remove_users']
+    permissions = BitField(flags=PERMISSION_FLAGS) # type: BitHandler
+
+    DEFAULT_STREAM_COLOR = "#c2c2c2"
```

Comment:
Why did you remove the Unicode designation (`u...`) from the `DEFAULT_STREAM_COLOR` constant?